src/apiCalls/api.js

```diff
@@ -0,0 +1,12 @@
+const getData = (endpoint) => {
+   return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${endpoint}`)
```

Comment:
I'm guessing you've done this already, but just in case - be sure to move this api call into it's own file.